environments/staging-aws-test/public.yml

```diff
@@ -0,0 +1,93 @@
+riak_backend: "leveldb"
+riak_ring_size: 64
+
+DATADOG_ENABLED: False
+elasticsearch_endpoint: '{{ groups.elasticsearch.0 }}'
+elasticsearch_cluster_name: 'staginges'
+elasticsearch_version: 1.7.6
+elasticsearch_download_sha256: 78affc30353730ec245dad1f17de242a4ad12cf808eaa87dd878e1ca10ed77df.
```

Comment:
nit: is the dot at the end is intended or a typo?

Reply:
Interesting! I hadn't noticed this at all (the whole file was just copied from staging), but searching for the same variable on other envs, I'm seeing that a lot of them have a trailing period. I'm not sure if that came from an initial error or not, but it doesn't seem to cause a problem. @NitigyaS do you know if that period (`.`) has a particular meaning, or if it's just a typo that ended up not causing a problem?